app/models/hackathon/gathering.rb

```diff
@@ -0,0 +1,9 @@
+module Hackathon::Gathering
+  extend ActiveSupport::Concern
+
+  included do
+    enum modality: {in_person: 0, online: 1, hybrid: 2}
```

Comment:
I think this should be a tag, and then if needed we can have `Hackathon#in_person?` which will check for that tag.

Reply:
Notes from today's one-on-one:
- `modality` will remain on `Hackathon` as an enum. This is because `Hackathon`s and `Tag`s are a one-to-many relationship, meaning that additional validation will need to be added to prevent a Hackathon from having both `in_person` and `online` tags.
- In addition, the current planned usage of tags do not require for them to be exposed to in the API. Adding `modality` as tags would require us to distinguish between public and private tags since `modality` needs to be exposed in the API.